File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_posHeight_int(string prompt);

int main(void)
{
    int h = get_posHeight_int("Height: "); //var for height
    //going row by row
    for (int i = 0; i < h; i++) 
    {
        //printing empty spaces before the wall
        for (int j = 1; j < h - i; j++)
        {
            printf(" ");
        }
        
        //printing the left part wall
        for (int w = 0; w <= i; w++)
        {
            printf("#");
        }
        
        //print blank space
        printf("  ");
        
        //printing the right part wall
        for (int w = 0; w <= i; w++)
        {
            printf("#");
        }
               
        //printing next line
        printf("\n");
    }
}

//making sure the height entered is between 1 and 8 inclusive
int get_posHeight_int(string prompt)
{
    int n;
    do
    {
        n = get_int("%s", prompt);
    }
    while (n < 1 || n > 8);
    return n;
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    x = get_int("Height: ")

    # accept only positive int no greater than 8
    if x > 0 and x < 9:
        break

for i in range(x):
    # printing white spaces on left
    print(" " * (x-i-1), end="")

    # printing left wall
    print("#" * (i + 1), end=" ")

    # printing middle space
    print(" ", end="")

    # printing right wall
    print("#" * (i + 1))
